etc/local.properties.example

```diff
@@ -72,6 +72,10 @@ testng.useDefaultListeners=false
 # etc/ivyconf.xml for more information.
 loci.resolver=omero-resolver
 
+# Controls how long CmdCallbackI based tests will wait
+# to finish.
```

Comment:
Milliseconds would be closest, though since this really only sets the scaling factor, the exact timeout is dependent on each test (though in the most common case it would be milliseconds).